src/main/webapp/script.js

```diff
@@ -327,10 +327,12 @@ function setupNavBarProfileSection() {
     .then((response) => response.json())
     .then((loginResponse) => {
       const profileLinks = $('#profileLinks');
-
+      const logLinks = $('#logLinks');
       if (loginResponse.isUserLoggedIn) {
         profileLinks.append($(
-          '<a class="nav-link text-light" href="/ProfilePage.html">Profile</a>'));
+        '<a class="nav-link text-light" href="/ProfilePage.html">Profile</a>'));
+        logLinks.append($(
+        '<a class="nav-link text-light" href="'+ loginResponse.LogoutURL + '">Logout</a>'));
       } else {
         profileLinks.append($(
         '<a class="nav-link text-light" href="'+ loginResponse.LoginURL + '">Login</a>'));
```

Comment:
should this use `logLinks` instead of `profileLinks` since its adding the login button?

Reply:
I tried using profileLinks. The buttons were getting stacked on top of each other instead of side by side

Reply:
oh weird. do you mean `logLinks`?

Reply:
I'm sorry- I confused login with the logout button. The Login button can still use profileLinks because it will just take the place of where the profile button shows up after users log in.

Reply:
no worries :) that sounds reasonable to me